src/main/java/core/basesyntax/dao/FruitReportDaoImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.dbreport.Report;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.Map;
+
+public class FruitReportDaoImpl implements FruitReportDao {
+
+    @Override
+    public void createReport(String fileName) {
+        String eol = System.getProperty("line.separator");

```

Comment:
<img width="515" alt="image" src="https://user-images.githubusercontent.com/81781479/202856548-922c2c2b-a91d-4f34-897d-27b540ea579d.png">


this method should return string
and FileWriter should write that string to file